packages/algob/test/internal/core/config/config-loading.ts

```diff
@@ -68,32 +68,36 @@ describe("config loading", function () {
     });
 
     it("should accept a relative path from the CWD", function () {
-      const config = loadConfigAndTasks({ config: "config.js" });
+      (async () => {
+        const config = await loadConfigAndTasks({ config: "config.js" });
 
-      if (!config.paths) {
-        assert.fail("Project was not loaded");
-      }
+        if (!config.paths) {
+          assert.fail("Project was not loaded");
+        }
 
-      assert.equal(
-        config.paths.configFile,
-        path.normalize(path.join(process.cwd(), "config.js"))
-      );
+        assert.equal(
+          config.paths.configFile,
+          path.normalize(path.join(process.cwd(), "config.js"))
+        );
+      })().catch((err) => console.log(err));
     });
 
     it("should accept an absolute path", function () {
-      const fixtureDir = getFixtureProjectPath("custom-config-file");
-      const config = loadConfigAndTasks({
-        config: path.join(fixtureDir, "config.js")
-      });
-
-      if (!config.paths) {
-        assert.fail("Project was not loaded");
-      }
-
-      assert.equal(
-        config.paths.configFile,
-        path.normalize(path.join(process.cwd(), "config.js"))
-      );
+      (async () => {
+        const fixtureDir = getFixtureProjectPath("custom-config-file");
+        const config = await loadConfigAndTasks({
+          config: path.join(fixtureDir, "config.js")
+        });
+
+        if (!config.paths) {
+          assert.fail("Project was not loaded");
+        }
+
+        assert.equal(
+          config.paths.configFile,
+          path.normalize(path.join(process.cwd(), "config.js"))
+        );
+      })().catch((err) => console.log(err));
```

Comment:
don't do this. You can make the function `async`:
```
 it("should accept an absolute path", async function () {
```
Mocha works well with async functions.

Reply:
same in other tests. 

But, if we will be ok, that in the condig, we will not use `await` then we don't need this changes.

Reply:
ok